test/integration/connect/envoy/run-tests.sh

```diff
@@ -424,7 +431,7 @@ function suite_setup {
 
     # pre-build the test-sds-server container
     echo "Rebuilding 'test-sds-server' image..."
-    docker build -t test-sds-server -f Dockerfile-test-sds-server .
+    docker build -t test-sds-server -f Dockerfile-test-sds-server test-sds-server
```

Comment:
This cuts down on how much random context needs to be shipped  during image construction.